test/chains/icon/localnet.go

```diff
@@ -327,7 +329,7 @@ func (c *IconLocalnet) SetupIBC(ctx context.Context, keyName string) (context.Co
 		return nil, err
 	}
 
-	client, err := c.getFullNode().DeployContract(ctx, c.scorePaths["client"], c.keystorePath, "ibcHandler="+ibcAddress)
```

Comment:
This can be reverted before we commit to main. It was needed when testing with a old client contract